plugin/src/main/java/io/jenkins/plugins/casc/yaml/YamlUtils.java

```diff
@@ -38,8 +32,8 @@
 
     public static final Logger LOGGER = Logger.getLogger(ConfigurationAsCode.class.getName());
 
-    public static Node merge(List<YamlSource> sources,
-        ConfigurationContext context) throws ConfiguratorException {
+    public static Node merge(List<YamlSource> sources, ConfigurationContext context,
+        MergeStrategy mergeStrategy) throws ConfiguratorException {
```

Comment:
Can we put `MergeStrategy` into `ConfigurationContext`?

Reply:
As suggested by @jetersen and @ JohnNiang, I removed the strategy parameter. Please help to review the [changes](https://github.com/jenkinsci/configuration-as-code-plugin/pull/1218/commits/9596269290654b8b13e9d47eb1cfbed61bbb08d9).

Reply:
Please fix checkstyle violation.